server/views/components/datePicker/datePicker.scss

```diff
@@ -0,0 +1,58 @@
+/*Datepicker overrides*/
```

Comment:
Doubt SASS will get compiled from here? Should go in `/assets/sass/...`

Reply:
Yes, agreed - make it a `_datepicker.scss` component and include in `application.sass`

Reply:
Will look into it - again, lifted from DPS and seems to work

Reply:
Actually the scss seems to indicate this might/should be a datepicker component - will look into this further on DPS - see what else i have to bring across to make this happen

Reply:
Sorted